scalableminds/webknossos
# Upload hybrid tracings as nml/zip

### URL of deployed dev instance (used for testing):
- https://uploadhybrid.webknossos.xyz

### Steps to test:
- upload single skeleton tracing, should still work
- upload zip with multiple skeleton tracings, should still work (result is one merged tracing)
- upload hybrid tracing (as generated by the code in #3025 ), should work
- upload invalid tracing files, should get understandable error message
- upload zip with multiple skeletons *and* a volume tracing, result should be hybrid with all of the stuff
- uploaad zip with multiple volumes, should get understandable error message

### Issues:
- contributes to #2987 

------
